content/docs/conditional-rendering.md

```diff
@@ -169,7 +169,7 @@ render() {
 }
 ```
 
-It can also be used for larger expressions although it is less obvious what's going on:
+它亦可以用在較大的 expression 上，雖然不是太直觀：
```

Comment:
How about `不是太明顯`?

Reply:
```suggestion
 它也可以被用在較複雜的表達式上
```

Reply:
@neighborhood999 I saw in [introducing JSX](https://github.com/reactjs/zh-hant.reactjs.org/blob/master/content/docs/introducing-jsx.md#%E5%9C%A8-jsx-%E4%B8%AD%E5%B5%8C%E5%85%A5-expression-embedding-expressions-in-jsx) it was kept as `expression` instead of `表達式`, so maybe to keep it consistent?

Reply:
@chloewlin the original text was `it is less obvious what's going on`, in my understanding it's similar to `less intuitive`, so seems `不是太明顯` was a bit different in meaning?